packages/editor/src/components/provider/index.native.js

```diff
@@ -28,6 +28,7 @@ import {
 	createBlock,
 } from '@wordpress/blocks';
 import { withDispatch, withSelect } from '@wordpress/data';
+import { store as editPostStore } from '@wordpress/edit-post';
```

Comment:
Let's avoid this cyclic dependency for `@wordpress/editor` package and leave it as a string for now.